docusaurus.config.js

```diff
@@ -38,7 +38,7 @@ const config = {
 
   scripts: [
     {
-      src: '/js/matomo.js',
+      src: process.env.LISK_DOC_URL ?? '/' + 'js/matomo.js',
```

Comment:
isn't it suppose to be something like
```suggestion
      src: `${process.env.LISK_DOC_URL ?? ''}/js/matomo.js`
```

Reply:
isn't this just an alternative way to concatenate strings?

Reply:
Ah actually I miss read what it's doing. All good